adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/WEB-INF/messages/base.properties

```diff
@@ -33,3 +33,4 @@ payment.method.issuer.selector=Please select issuer
 payment.method.terminal.selector=Select your terminal
 
 checkout.summary.spinner.message=Please wait while your payment is processed. Do not click back or refresh the page.
+checkout.summary.component.payment=Provide your account data to finalize your payment
```

Comment:
This error is not cleat like what is expected from user. what is account data?

Reply:
It's not an error, it's the message shown on the summary page before the component form to submit the payment.
Changed the message to make clear it's the MB Way data that need to be provided.